src/multiplayer-servers/getting-started/sidebar.json

```
@@ -31,6 +31,10 @@
       "text": "Setup your Environment",
       "link": "/getting-started/setup-your-environment"
     },
+    {
+      "text": "Setup 3rd-Party Authentication",
```

```suggestion
      "text": "Setup Third-Party OAuth Integration",
```

IMO the navigation item text should match the documentation title.

---

Updated.